eth-contracts/Makefile

```diff
@@ -0,0 +1,7 @@
+test:
```

Comment:
Do not use both makefile and package.json to set commands.
You can write this commands in package.json.